src/parser/semantic_check.c

```diff
@@ -7984,7 +7985,9 @@ pt_check_create_view (PARSER_CONTEXT * parser, PT_NODE * stmt)
 
       /* TODO This seems to flag too many queries as view specs because it also traverses the tree to subqueries. It
        * might need a pre_function that returns PT_STOP_WALK for subqueries. */
-      result_stmt = parser_walk_tree (parser, crt_qry, pt_set_is_view_spec, NULL, NULL, NULL);
+      do_not_replace_orderby = true;
+      result_stmt =
+	parser_walk_tree (parser, crt_qry, pt_set_is_view_spec, (void *) &do_not_replace_orderby, NULL, NULL);
```

Comment:
Before calling pt_semantic_check() at the bottom instead of the modified contents here
Can't we just add the code as if we modified the pt_check_vclass_query_spec() function?
In this way, it seems that the modification of the pt_set_is_view_spec() function is not necessary.

```
**(void) parser_walk_tree(parser, crt_qry, pt_set_do_not_replace_orderby, NULL, NULL, NULL);**
result_stmt = pt_semantic_check(parser, crt_qry);
```

Reply:
To reduce calls to parser_walk_tree (), I used pt_set_is_view_spec.

Reply:
Yes, I understand.
Then, how about removing the pt_set_do_not_replace_orderby() function and modifying the code on line 13494 in the pt_validate_query_spec() function as follows?

s = parser_walk_tree(parser, s, pt_set_is_view_spec, NULL, NULL, NULL);
==> 
bool do_not_replace_orderby = true;
s = parser_walk_tree(parser, s, pt_set_is_view_spec, (void *) &do_not_replace_orderby, NULL, NULL);

Reply:
I fixed it. thanks.